spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -1783,6 +1783,13 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde {
         if (aggregateExpressions.isEmpty) {
           val hashAggBuilder = OperatorOuterClass.HashAggregate.newBuilder()
           hashAggBuilder.addAllGroupingExprs(groupingExprs.map(_.get).asJava)
+          val attributes = groupingExpressions.map(_.toAttribute) ++ aggregateAttributes
+          val resultExprs = resultExpressions.map(exprToProto(_, attributes))
+          if (resultExprs.exists(_.isEmpty)) {
+            emitWarning(s"Unsupported result expressions found in: ${resultExpressions}")
+            return None
+          }
```

Comment:
Even `aggregateExpressions` is empty, we still need to assign result expressions. This is the latest bug I found during fixing TPCDS query failures in SortMergeJoin work.